members/views.py

```diff
@@ -1,9 +1,27 @@
-from django.contrib.auth import get_user_model
+import bcrypt
+from rest_framework import status
 from rest_framework.generics import CreateAPIView
-from .serializers import SignupSerializer
-from members import serializers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django.contrib.auth.hashers import check_password
+import jwt
+from .models import User
+from .serializers import UserSerializer
+from secret import SECRET
+
 
 # Create your views here.
 class SignupView(CreateAPIView):
-	model = get_user_model()
-	serializer_class = SignupSerializer
+	model = User
+	serializer_class = UserSerializer
+
+
+class SigninView(APIView):
+	def post(self, request, format=None):
+		user = User.objects.get(email=request.data["email"])
+		if not user:
+			return Response(" 존재하지 않는 이메일 ", status=status.HTTP_400_BAD_REQUEST)
+		if not bcrypt.checkpw(request.data["password"].encode('utf-8'), user.password.encode('utf-8')):
+			return Response(" 유효하지 않은 비밀번호 ", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
이것도 역시 인증이 되지 않은 것이기에 `401 Unauthorized`가 더 어울리는 것 같습니다.

```suggestion
			return Response(" 유효하지 않은 비밀번호 ", status=status.HTTP_401_UNAUTHORIZED)
```